lib/libvarnish/vsub.c

```diff
@@ -38,10 +38,8 @@
 #include <stdlib.h>		// Solaris closefrom(3c)
 #include <string.h>
 #include <unistd.h>		// BSD/Linux close_range(2)
-#ifdef HAVE_WORKING_CLOSE_RANGE
-#elif HAVE_CLOSEFROM
-#else
-#  include <dirent.h>
+#ifndef HAVE_CLOSEFROM
+#include <dirent.h>
 #endif
```

Comment:
Please bring back the `#  include` indentation, it's nested inside an `#ifdef`.

Reply:
Indented